Source/iOS/UIColor+Hue.swift

```diff
@@ -173,6 +173,30 @@ public extension UIColor {
       return a
     }
   }
+
+	var hueComponent : CGFloat {
+		get {
```

Comment:
It is computed property, I don't think you need `get` here

Reply:
The same as before, I know you don't need the get, but left it following the coding convention on the same extension you used for redComponent.